clinica/iotools/bids_utils.py

```diff
@@ -614,6 +626,15 @@ def compute_new_subjects(original_ids, bids_ids):
     return to_return
 
 
+def years_to_months(years):
+    """Converts a number of years in decimal to a number of months rounded."""
+    from math import trunc
+
+    nb_years = trunc(years)
+    nb_months = (years - nb_years) * 12
+    return round(nb_months + nb_years * 12)
```

Comment:
Isn't it enough to do `round(years * 12)`? For instance, `1.5 y * 12 m / y = 18 m`.

Or am I missing something?